schema/Mutation.graphql

```diff
@@ -36,4 +36,16 @@ type Mutation {
     Authentication is required
     """
     refreshCredentials: String!
+
+    """
+    Create a mutation to create new lives
+    """
+    createLife(
+        firstName: String!
+        lastName: String!
+        title: String!
+        birthday: DateTime!
+        description: String!
+        hobbies: [String!]!
```

Comment:
It will be more efficient if you wrap this in an object

Reply:
done